src/datavalues.jl

```diff
@@ -6,6 +6,9 @@ unwrap(x::DataValue) = isna(x) ? missing : DataValues.unsafe_get(x)
 datavaluetype(::Type{T}) where {T <: DataValue} = T
 datavaluetype(::Type{Union{T, Missing}}) where {T} = DataValue{T}
 datavaluetype(::Type{Missing}) = DataValue{Union{}}
+scalarconvert(T, x) = convert(T, x)
```

Comment:
Would be nice to add a comment pointing to the issue, since we should be able to get rid of this at some point.